src/types/venues.ts

```diff
@@ -11,7 +11,6 @@ import { Poster } from "./posters";
 import { Quotation } from "./Quotation";
 import { Room } from "./rooms";
 import { Table } from "./Table";
-import { UpcomingEvent } from "./UpcomingEvent";
```

Comment:
I believe everything that is related to the `UpcomingEvent` type is legacy. Can you remove that?

Reply:
It's being used in `UpcomingTickets` component. Can we confirm that it's not necessary anymore?

Reply:
99% - `UpcomingTickets` are legacy. You can remove them. Confirming.
(clarified with @mike-lvov )